src/cipher/encryptingstorage/__init__.py

```diff
@@ -147,10 +146,10 @@ def invalidateCache(self):
         """
         return self.db.invalidateCache()
 
-    def invalidate(self, transaction_id, oids, version=''):  # the version arg is still present in ZODB 4.3.1, but is removed in ZODB 5.2.3
+    def invalidate(self, transaction_id, oids, version=''):
         """ For IStorageWrapper
         """
-        return self.db.invalidate(transaction_id, oids, version)
+        return self.db.invalidate(transaction_id, oids)
```

Comment:
I think let's remove the `version` argument from the method signature as well.  Or at least `assert version == ''`.

Reply:
This will remove the support for ZEO / ZODB 4 if you do that.

Reply:
The `assert` version wouldn't do that, as long as you don't use versions. 

(I don't believe anyone has used ZODB versions in the last decade.)